certification/pyxis/types.go

```diff
@@ -4,6 +4,7 @@ import "github.com/redhat-openshift-ecosystem/openshift-preflight/certification/
 
 type CertImage struct {
 	ID                     string       `json:"_id,omitempty"`
+	Links                  Links        `json:"_links"`
```

Comment:
This should be omitempty.

Reply:
I'm going to remove this struct, it does not buy us anything. I thought that these values were populated _once_ say an `rpm manifest` or `results` were created, but that is not the case, these values are populated instantly even if `rpm manifest` or `results` haven't been created yet.